Source/HYPFormsCollectionViewDataSource.m

```diff
@@ -511,6 +511,7 @@ - (void)processTarget:(HYPFormTarget *)target
             NSArray *deletedIndexPaths = [self.formsManager hideTargets:@[target]];
             [self deleteItemsAtIndexPaths:deletedIndexPaths];
         } break;
+        case HYPFormTargetActionClear:
```

Comment:
But if we don't have a break, doesn't it just hoop down to the next and use update. See last commit for
my way of thinking.